torch/quantization/fx/qconfig_utils.py

```diff
@@ -13,6 +13,21 @@
 QConfigAny = Union[torch.quantization.QConfig,
                    torch.quantization.QConfigDynamic, None]
 
+def add_device_to_obs_ctr_in_qconfig(qconfig_dict, device='cpu'):
+    if '' in qconfig_dict.keys() and isinstance(qconfig_dict[''], torch.quantization.QConfig):
+        factory_kwargs = {'device': device}
+        new_activation = qconfig_dict[''].activation.with_args(factory_kwargs=factory_kwargs)
```

Comment:
Where is the device attr used in the observer?

Reply:
Also this assumes all observers will pass in the factory_kwargs to the buffers that it creates. Would this break if the user has their own observer code or we have buffers that don't use the kwargs?

Reply:
it wouldn't break, it just wouldn't actually do anything in those cases.All that would happen is that if the user doesn't set up their custom observer correctly, it just wouldn't work with gpu quantization.

It seems fairly low risk since that argument seems entirely unused as is. factory_kwargs is specifically intended to handle device info so anything that utilizes it would already need to be able to handle a device argument.

Reply:
I'm removing these changes in any case, it seems unclear as to how the team wants to solve this issue and its not actually a blocker for this PR